contracts/GroupVoteLocker.sol

```diff
@@ -0,0 +1,331 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.4;
+
+import "./BaseVoteLocker.sol";
+import "OpenZeppelin/openzeppelin-contracts@02fcc75bb7f35376c22def91b0fb9bc7a50b9458/contracts/utils/math/SafeCast.sol";
+import "OpenZeppelin/openzeppelin-contracts@02fcc75bb7f35376c22def91b0fb9bc7a50b9458/contracts/utils/Strings.sol";
+import "./console.sol";
+
+contract GroupVoteLocker is BaseVoteLocker {
```

Comment:
There are a lot of `public` methods in this that can probably be `external`. Should make the contract deploy size smaller.